File: sample/src/main/java/pl/droidsonroids/gif/sample/GifTextViewFragment.java
package pl.droidsonroids.gif.sample;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class GifTextViewFragment extends BaseFragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.compound_drawables, container, false);
    }
}

File: sample/src/main/java/pl/droidsonroids/gif/sample/GifTextViewFragment.kt
package pl.droidsonroids.gif.sample

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

class GifTextViewFragment : Fragment() {
	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? =
			inflater.inflate(R.layout.compound_drawables, container, false)
}
